.github/workflows/test.yml

```diff
@@ -144,14 +144,17 @@ jobs:
         path: build/reports/tests
 
     - name: "Notify QA Slack"
-      if: steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging'
+      if: ${{ always() }}
+#      if: steps.set-env-step.outputs.test-env == 'alpha' || steps.set-env-step.outputs.test-env == 'staging'
```

Comment:
remove commented code, here and below.

Reply:
@jaycarlton thanks for the review - this is only commented out so I can force notifications on each test run, instead of only the alpha/staging runs. When this is ready for review, I will move it out of draft!

Reply:
Whoops. I saw the notification in Slack. thought those were only for actual reviews now.